File: python_Programmers/1단계/x만큼간격이있는n개의숫자.py
def solution(x, n):
    answer = []
    for i in range(1, n+1):
        answer.append(x * i)
    return answer


# 모범풀이
def solution2(x, n):
    return [x*(i+1) for i in range(n)]

x = -4
n = 2
print(solution(x, n))
print(solution2(x, n))

File: java/programmers/1단계/src/x만큼간격이있는n개의숫자.java
import java.util.Arrays;

public class x만큼간격이있는n개의숫자 {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];
        long sum = x;
        for (int i = 0; i < n; i++) {
            answer[i] = sum;
            sum += x;
        }
        return answer;
    }

    public static void main(String[] args) {
        x만큼간격이있는n개의숫자 sol = new x만큼간격이있는n개의숫자();
        int x = 2;
        int n = 5;
        System.out.println(Arrays.toString(sol.solution(x, n)));
    }
}
